doc/1.manual/x5.htm

```diff
@@ -28,14 +28,25 @@ <H3> <A id="s5.1"> 5.1. Release notes </A> </H3>
 <P> ------------------ 0.55-0 ------------------------------
 
 <P> Many documentation updates, thanks to Alexandre Porres, Ben Wesch, and
-others.
+others. Most notably several updates and edits were made to Pd's HTML manual,
+specially a new section on 'advanced editing techniques' (aka 'intelligent
+patching').
 
-<P> 64-bit soundfile reading and writing support by Dan Wilcox.
+<P> The [oscparse] object was updated by Porres to allow numeric addresses
+via a new '-n' flag and a right outlet was added to output split point
+between address and message.
+
+<P> Minor improvements to the [file] object by IOhannes, such as properly
+expanding '~' to home directory and allowing [file which] to also search
+paths relative to the parent patch.
+
+<P> 64-bit soundfile reading and writing support by Dan Wilcox for [readsf~],
+[writesf~] and [soundfiler].
 
 <P> Many bug fixes and code-level updates and improvements, notably to libpd and
-for emscripten support, thanks as usual to Iohanes, Dan, and Christof.  The
-emscripten support, in particular, should someday allow Pd to run in a web
-browser.
+for emscripten support thanks to Claude Heiland-Allen and IOhannes, Dan, and
+Christof. The emscripten support, in particular, should someday allow Pd to run
+in a web browser.
 
```

Comment:
Maybe you could also mention the new empty `[set(` method for `[throw~]` and `[receive~]`. `sys_getfunbyname` is probably worth mentioning as well.